constants.js

```diff
@@ -62,6 +62,8 @@ const checksWhitelist = {
   'oppia-android': {
     [openEvent]: [claCheck],
     [reopenEvent]: [],
+    [openEventGithubActions]: [claCheckGithubAction],
+    [reopenEventGithubActions]: [claCheckGithubAction],
```

Comment:
Why are these added?

Reply:
This is meant to enable The CLA Check GitHub action for Oppia-Android